build.gradle.kts

```diff
@@ -32,12 +16,3 @@ detekt {
         xml.enabled = true
     }
 }
-
-subprojects {
-    repositories {
-        mavenCentral()
-        google()
-    }
-
-    group = Description.mavenGroup
```

Comment:
Group is set in [publishing convention](https://github.com/KakaoCup/Kakao/pull/109/files#diff-3d192919d82da09de1761cb1e097e6b66129795db31cc40787b4b276786525dbR18) now